_includes/apple-touch-icons.md

```diff
@@ -0,0 +1,9 @@
+{% comment -%}
+TODO: Figure out how to use front matter with proper arrays instead of split
```

Comment:
Why would we want to use front matter for this? Having them all defined here lets us have all the sizes defined in one place instead of redefining them in the front matter on every page, or am I misunderstanding something for that?

Reply:
I was thinking front matter inside the `apple-touch-icons.md` file itself would be nice. But I guess we will have to wait for jekyll/jekyll#8228 to be solved first.

Reply:
Ah did not think about that.
No wonder, you propsed that feature 19 hours ago! 😄